### PR TITLE
Devex/use dockerhub cassandra container

### PR DESCRIPTION
**Pull request by SerialVelocity**
---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/1102)

### CODE REVIEW

File: atlasdb-cassandra-integration-tests/src/test/resources/docker-compose.yml [hunk 1]
```diff
@@ -2,6 +2,6 @@ version: '2'
 
 services:
   cassandra:
-    image: com.palantir.docker/atlas-cassandra:2.2-v0.1
+    image: palantirtechnologies/docker-cassandra-atlasdb:latest
```
Comment (gbrova) [inline atlasdb-cassandra-integration-tests/src/test/resources/docker-compose.yml]: Consider extracting a specific version instead of latest

Comment (SerialVelocity) [inline atlasdb-cassandra-integration-tests/src/test/resources/docker-compose.yml]: There are no versions. :)

Comment (SerialVelocity) [inline atlasdb-cassandra-integration-tests/src/test/resources/docker-compose.yml]: (maybe we should tag one in the other repo)

Comment (gbrova) [inline atlasdb-cassandra-integration-tests/src/test/resources/docker-compose.yml]: That's surprising. We should definitely tag them next time we change the other repo, if not sooner. I wouldn't block on it for this PR.

**Pull request merged**